plotly/plotly/plotly.py

```diff
@@ -869,7 +868,7 @@ def _fill_in_response_column_ids(cls, request_columns,
         for req_col in request_columns:
             for resp_col in response_columns:
                 if resp_col['name'] == req_col.name:
-                    req_col.id = '{0}/{1}'.format(grid_id, resp_col['uid'])
+                    req_col.id = '{0}:{1}'.format(grid_id, resp_col['uid'])
```

Comment:
Yeah, it wasn't working until I made the switch. Here's one example that didn't work before the change that worked after:
```
c1 = Column([6, 6, 6, 5], 'pillow')
c2 = Column(['a', 'b', 'c', 'd'], 'couch2')
g = Grid([c1, c2])

py.grid_ops.upload(g, 'pillow_and_couch_grid222', auto_open=False)
url = py.plot([Scatter(xsrc=g[0], ysrc=g[1])], auto_open=False, filename='222pillow222')
```

I'm curious about something though...when plotly makes the request to V2 for the upload of the grid, prior to that, the local grid has blank `id`s (eg. `grid[0].id` returns `""` ) and after the request, your local grid has the proper filled in id eg. "AdamKulidjian:1782:u3fg1"

How is my local grid getting updated if only a url after `py.upload()` is run?